corehq/apps/smsbillables/models.py

```diff
@@ -395,9 +395,9 @@ def _get_usage_fee(cls, domain, direction):
         return usage_fee
 
     @classmethod
-    def _get_twilio_charges(cls, backend_message_id, direction, couch_id):
+    def _get_twilio_charges(cls, backend_message_id, backend_instance, direction, couch_id):
         def _get_twilio_client():
-            twilio_backend = SQLTwilioBackend.get_global_backends_for_this_class(SQLTwilioBackend.SMS).get()
+            twilio_backend = SQLTwilioBackend.objects.get(couch_id=backend_instance)
```

Comment:
this should be:

`twilio_backend = SQLMobileBackend.load(backend_instance, api_id=SQLTwilioBackend.get_api_id(), is_couch_id=True, include_deleted=True)`